llvm/tools/llvm-split/llvm-split.cpp

```diff
@@ -70,6 +76,64 @@ static cl::opt<std::string>
     MCPU("mcpu", cl::desc("Target CPU, ignored if --mtriple is not used"),
          cl::value_desc("cpu"), cl::cat(SplitCategory));
 
+cl::opt<IRSplitMode> SYCLSplitMode(
+    "sycl-split", cl::desc("module split mode"), cl::Optional,
```

Comment:
I think the `cl::desc` here could be more human-friendly, like the other options in the same file. It should start with a capital letter, and might be best if it specifically mentions `SYCL` in the description, too.

Shouldn't the options be `static`?

Reply:
Looks good, thanks